content/models.py

```diff
@@ -159,3 +159,50 @@ class DonateSection(models.Model):
 
 	def __str__(self):
 		return self.title
+
+class MenuNavbar(models.Model):
+
+    link = models.CharField((u'link'), max_length=100, )
+    title = models.CharField((u'title'), max_length=250)
+    parent = models.ForeignKey('self', blank=True, null=True, related_name='child')
+
+    def __str__(self):
+    	return self.title
+
+    class Meta:
+        verbose_name_plural = 'MenuNavbars'
+        ordering = ['title']
+
+    def __unicode__(self):
+        p_list = self._recurse_for_parents(self)
+        p_list.append(self.title)
+        return self.get_separator().join(p_list)
+
+    def _recurse_for_parents(self, cat_obj):
+        p_list = []
+        if cat_obj.parent_id:
+            p = cat_obj.parent
+            p_list.append(p.title)
+            more = self._recurse_for_parents(p)
```

Comment:
branch off of this branch, so you can try without ruining the work you already did